lambda_service/insert_data.py

```diff
@@ -56,36 +42,49 @@ def sendToSubscribers(data):
     dataToSend = {"messages": [data]}
     for connectionID in connections:
         try: 
-            connectionResponse = _send_to_connection2(gatewayApi, connectionID, dataToSend, os.getenv('WSS_URL'))
-            print('connection response: ')
-            print(json.dumps(connectionResponse))
+            connectionResponse = _send_to_connection(
+                    gatewayApi, connectionID, dataToSend, os.getenv('WSS_URL'))
+            logger.info((
+                f"connection response: "
+                f"{json.dumps(connectionResponse)}")
+            )
         except gatewayApi.exceptions.GoneException:
-            print(f"Failed sending to {connectionID} due to GoneException. Removing it from the connections table.")
+            error_msg = (
+                f"Failed sending to {connectionID} due to GoneException. "
+                "Removing it from the connections table."
+            )
+            logger.exception(error_msg)
             _remove_connection(connectionID)
             continue
 
-def main2(event, context):
-    sendToSubscribers("hello")
 
-def main(event, context):
+def handle_s3_object_created(event, context):
+
+    logger.info(event)
```

Comment:
Consider giving this log message some context, just so if you see the log message you know (roughly) where in the codebase the log was emitted.